src/parse.rs

```diff
@@ -72,8 +72,8 @@ pub fn parse_file(source_path: &Path, lex_path: &Path, yacc_path: &Path) -> Resu
     Ok(gen_bytecode(&pt, &grm, &input))
 }
 
-#[derive(Debug)]
-enum Instr {
+#[derive(Debug, Clone)]
```

Comment:
Are you sure you want to `clone` an instruction, given that some variants (e.g. `PUSH_STR`) could have arbitrarily sized strings in them? That could make `clone`ing somewhat expensive.

Reply:
No I don't want to clone it. I think i mistakenly left this in after cloning it to fix a somewhat related compiler error down the line. This should have been a reference, thanks for the spot.